src/pages/Notifications/Overview/Page.tsx

```diff
@@ -261,9 +263,8 @@ export const NotificationsOverviewPage: React.FunctionComponent = () => {
                 linkTitle="Go to Notification Preferences"
                 linkTarget={`${
                   isBeta() ? '/preview' : ''
-                }/${getBundle()}/notifications/user-preferences`}
-                expandableContent="This service allows you to opt-in and out of receiving notifications. Your Organization
-                            Administrator has configured which notifications you can or can not receive in their Settings."
+                }/${getBundle()}/notifications/my-notification-preferences`}
```

Comment:
the url should still be `/notifications/user-preferences`

Reply:
good catch!